tests/RobotFramework/tests/customizing/data_path_config.robot

```diff
@@ -36,3 +50,35 @@ Custom Teardown
     Execute Command    sudo rm -rf /var/test
     Execute Command    sudo tedge config unset data.path
     Get Logs
+
+Create child device with firmware support
```

Comment:
It's kind of a `boot strap` process for supporting the firmware ops support right?